.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,49 @@
+name: Run PR checks
+run-name: Checks for PR#${{ github.event.pull_request.number }}
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
This is fine for now, but we will probably want to run this on any branch eventually.

Reply:
Makes sense. I wasn't sure what kind of branching strategy you had in mind, but this is trivial to update as necessary.